runtime/src/fees/payment.rs

```diff
@@ -95,3 +101,50 @@ fn org_payer_account(author: AccountId, org_id: &OrgId) -> AccountId {
         None => author,
     }
 }
+
+#[cfg(test)]
+mod test {
+    use core::convert::TryFrom;
+    use frame_support::traits::Currency;
+    use sp_core::{crypto::Pair, ed25519};
+    use sp_runtime::BuildStorage;
+
+    use crate::{Balances, GenesisConfig};
+
+    use super::*;
```

Comment:
`nit`

I recall us talking to first have the local imports and then the libraries. Doing as suggested below also makes these imports a bit more compact in terms of vertical space.

```suggestion
    use super::*;
    use crate::{Balances, GenesisConfig};
    
    use core::convert::TryFrom;
    use frame_support::traits::Currency;
    use sp_core::{crypto::Pair, ed25519};
    use sp_runtime::BuildStorage;    
```